spec/models/user_spec.rb

```diff
@@ -61,8 +61,17 @@
       let!(:new_account) { create(:account_sequence, active: false, user: user) }
 
       before { user.change_current_account_to(new_account.id) }
-      
+
       it { expect(user.current_account).to eq new_account }
     end
   end
+
+  context '#current_account' do
+    let!(:user) { create(:user) }
+    before { user.accounts.destroy_all }
+
+    context 'when not have account, should be create default account' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.